.circleci/config.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           name: Link the module being tested to the samples.
           command: |
             cd samples/
-            npm link ../
+            npm link @google-cloud/bigtable
```

Comment:
I am confused :) `npm link ../` will make sure `npm install` is run on the parent directory, AND do the linking.  Why switch to the named module?

Reply:
Yeah, I thought so as well. The error persists with any configuration I've tried so far (including the one that was in the config.yml before I made any changes).